x-pack/plugins/event_log/server/es/cluster_client_adapter.ts

```diff
@@ -120,19 +125,38 @@ export class ClusterClientAdapter<TDoc extends { body: AliasAny; index: string }
     }
   }
 
-  public async doesIlmPolicyExist(policyName: string): Promise<boolean> {
+  public async unlinkIlmPolicyFromOutdatedIndices(outdatedLinkedIndices: string[]): Promise<void> {
+    const indexTargets = (outdatedLinkedIndices ?? []).join(',');
+    const request = {
+      method: 'POST',
+      path: `${indexTargets}/_ilm/remove`,
+    };
+    try {
+      const esClient = await this.elasticsearchClientPromise;
+      await esClient.transport.request(request);
+    } catch (err) {
+      this.logger.error(
```

Comment:
Logging the error but not throwing because I don't think this should block the rest of the event log bootstrapping.